### ISSUE TITLE
SystemCallErrorNumber accepts Number or signal names

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/systemdlint#38]
https://www.freedesktop.org/software/systemd/man/systemd.exec.html#SystemCallErrorNumber=

currently this false positive is reported
```
rootfs/etc/systemd/system/sysinit.target.wants/systemd-timesyncd.service :42:1 - SystemCallErrorNumber=EPERM - [systemdlint.systemdlint.InvalidValue]
```
